libs/smartsync.js

```diff
@@ -189,10 +189,11 @@
     // A __local__ boolean field is added automatically on save
     // Index are created for keyField and __local__
     //
-    Force.StoreCache = function(soupName, additionalIndexSpecs, keyField) {
+    Force.StoreCache = function(soupName, additionalIndexSpecs, keyField, isGlobalStore) {
```

Comment:
Optional new argument to tie a StoreCache to the global store instead of the regular smartstore